src/Microsoft.ML.Data/Transforms/ExtensionsCatalog.cs

```diff
@@ -9,24 +9,24 @@
 
 namespace Microsoft.ML
 {
-    public sealed class SimpleColumnInfo
+    public sealed class ColumnOptions
```

Comment:
>ColumnOptions [](start = 24, length = 13)

SimpleColumnOptions

Reply:
It was intentional, why do you think that `SimpleColumnOptions` is better?
I thought that for consistency it was better like this.